src/components/Photo.css

```
@@ -0,0 +1,4 @@
+img {
```

```suggestion
.img {
```
Without the dot, this css was targeting the img tag instead of a class called 'img'. Interesting things happen when you change the code above.